deepchem/models/torch_models/layers.py

```diff
@@ -1286,3 +1286,62 @@ def forward(
                               (1, M_nbrs, 1))
     # Shape (N_atoms, M_nbrs)
     return torch.sum((tiled_coords - nbr_coords)**2, dim=2)
+
+
+class WeightedLinearCombo(nn.Module):
+  """
+  Compute a weighted linear combination of input layers, where the weight variables are trained.\
+
+  Usage:
```

Comment:
The formatting here looks a little off. We use numpydoc format for our docstrings (https://numpydoc.readthedocs.io/en/latest/format.html)

Reply:
My bad, got mixed up for a second.
Should be fixed, sorry about that.

Reply:
Looks like I needed some sleep.
I'm working on fixing the `Examples` section.

I have a question and I can't seem to find the answer on Google so I'm hoping somewhere here knows why.
I marked the result with `#random` but it still seems to give me a failed example with "Expected" and "Got".

Reply:
I'm stuck on one thing regarding the `Examples` section.
The output is inherently random so I marked the output with `#random` but `doctest` still fails because it got a different result than it expected (which is to be expected).
I haven't used `doctest` or the numpydoc format before and Google didn't make me much wiser.
I'm sure it's a simple fix so I'm hoping someone here knows why.

Reply:
Should be fixed now.